pixy/consumer.go

```diff
@@ -0,0 +1,543 @@
+package pixy
+
+import (
+	"fmt"
+	"sort"
+	"sync"
+	"time"
+
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/mailgun/kazoo-go"
```

Comment:
Should we switch this to wvanbergen/kazoo-go now that your patch has been merged?